Cargo.toml

```diff
@@ -10,10 +10,10 @@ keywords = ["twilio","rust"]
 edition = "2018"
 
 [dependencies]
-base64 = "0.12"
+base64 = "0.13.0"
 headers = "0.3.2"
-hyper = "0.13"
-hyper-tls = "0.4"
+hyper = { version = "0.14.4", features = ["full"] }
```

Comment:
And again here, should you require the minimum features.. Does this really need an http server with the `server` feature?

https://docs.rs/hyper/0.14.5/hyper/index.html#optional-features

Reply:
same here, will back to you, when I tested it